_pkgdown.yml

```diff
@@ -1,4 +1,5 @@
 template:
+  ganalytics: "UA-115082821-1"
```

Comment:
Does this actually do anything with the tidytemplate template?

Reply:
Oh, you think it's overridden by the UA in the tidytemplate footer? The way it keeps telling me to check is with this Tag Assistant extension, which doesn't work on local previews…so I honestly can't tell. I was missing the trailing `-1` on the dplyr one deployed, so I don't have a dummy version with which to check.